packages/webapp/src/containers/Setup/SetupSubscription/SetupSubscription.tsx

```diff
@@ -7,25 +6,27 @@ import { SubscriptionPlansSection } from './SubscriptionPlansSection';
 import withSubscriptionPlansActions from '../../Subscriptions/withSubscriptionPlansActions';
 import styles from './SetupSubscription.module.scss';
 
+// TODO: use typescript https://github.com/NuroDev/lemonsqueezy.ts/tree/main/src/modules/subscription#-subscription
```

Comment:
@abouolia kindly make the lemonsqueezy implementation type-safe. I had to disable this to get Vite to run. I can't test this functionality so I don't want to try to restore the functionality